### ISSUE TITLE
php-filter module is required and not listed in admin manual (NC21, NGINX, PHP8.0.2)

### ISSUE THREAD

**Issue by squid-f** [nextcloud/documentation#6235]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1.Install nginx and php-fpm (php8.0.2)
2.Install php modules per admin manual
3.Create the database
4.Download nextcloud
5.Launch the installation wizard, create the admin user and enter database credentials. 
6.Internal error will be sent because of missing module php-filter (refer to https://github.com/nextcloud/server/issues/25627)

### Expected behaviour
Pass the installation wizard step

### Actual behaviour
Internal error because of missing php-filter module.
For some reason, it was implicitly installed with php7.3 and php8.0.0 with my distro (Mageia7 and Mageia 8 beta).
It is not listed in the admin manual that this module is mandatory; I would add it.
I have spent hours before understanding the reason why I was not passing the installation wizard step

### Server configuration

**Operating system:** 5.10.19-desktop-1.mga8 x86_64 - MAGEIA 8

**Web server:** nginx 1.18

**Database:** postgresql 13.1 or mariaDB 10.5.8

**PHP version:** 8.0.2

**Nextcloud version:** 21.0.

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh

**Where did you install Nextcloud from:** nextcloud.com download page

**Comment by acsfer** [comment]
Filter extension is (should be) enabled by default: https://www.php.net/manual/en/filter.installation.php

**Comment by squid-f** [comment]
> Filter extension is (should be) enabled by default: https://www.php.net/manual/en/filter.installation.php

Hi. If I don't install php-filter explicitely, I don't get it by installing php-fpm for instance. I use NGINX server, so, I don't install apache-mod_php.
Which dependency is it from?
Should I file a bug report to Mageia distro dev?

**Comment by HouraisanNEET** [comment]
Reproduced on FreeBSD 13.0 with PHP 7.4.16. However there is even no section about installation on FreeBSD so I don't know where should I add it.
Installing ``php74-filter`` solves it.

### Server Log
<details>

```json
{
  "reqId": "TQ6VOjS2JbT2BzXheZVB",
  "level": 3,
  "time": "2021-04-29T19:07:58+00:00",
  "remoteAddr": "127.0.0.1",
  "user": "--",
  "app": "index",
  "method": "POST",
  "url": "/nextcloud/index.php",
  "message": {
    "Exception": "Error",
    "Message": "Call to undefined function OC\\Http\\Client\filter_var()",
    "Code": 0,
    "Trace": [
      {
        "file": "/usr/local/www/nextcloud/lib/private/Http/Client/Client.php",
        "line": 235,
        "function": "preventLocalAddress",
        "class": "OC\\Http\\Client\\Client",
        "type": "->",
        "args": [
          "https://127.0.0.1:25252/nextcloud/data/htaccesstest.txt",
          []
        ]
      },
      {
        "file": "/usr/local/www/nextcloud/lib/private/legacy/OC_Util.php",
        "line": 1267,
        "function": "get",
        "class": "OC\\Http\\Client\\Client",
        "type": "->",
        "args": [
          "https://127.0.0.1:25252/nextcloud/data/htaccesstest.txt"
        ]
      },
      {
        "file": "/usr/local/www/nextcloud/lib/private/Setup.php",
        "line": 235,
        "function": "isHtaccessWorking",
        "class": "OC_Util",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\AllConfig"
          }
        ]
      },
      {
        "file": "/usr/local/www/nextcloud/core/Controller/SetupController.php",
        "line": 58,
        "function": "getSystemInfo",
        "class": "OC\\Setup",
        "type": "->",
        "args": []
      },
      {
        "file": "/usr/local/www/nextcloud/lib/base.php",
        "line": 932,
        "function": "run",
        "class": "OC\\Core\\Controller\\SetupController",
        "type": "->",
        "args": [
          "*** sensitive parameters replaced ***"
        ]
      },
      {
        "file": "/usr/local/www/nextcloud/index.php",
        "line": 37,
        "function": "handleRequest",
        "class": "OC",
        "type": "::",
        "args": []
      }
    ],
    "File": "/usr/local/www/nextcloud/lib/private/Http/Client/Client.php",
    "Line": 188,
    "CustomMessage": "--"
  },
  "userAgent": "Mozilla/5.0 (X11; FreeBSD amd64; rv:78.0) Gecko/20100101 Firefox/78.0",
  "version": "21.0.1.1"
}
```
</details>

**Comment by kesselb** [comment]
https://github.com/nextcloud/documentation/blob/master/admin_manual/installation/source_installation.rst would be the right file to add it.

**Comment by HouraisanNEET** [comment]
> https://github.com/nextcloud/documentation/blob/master/admin_manual/installation/source_installation.rst would be the right file to add it.

Added a line. Though it's mainly for Linux as the title implies, but most of the steps can be applied on FreeBSD with proper changes.

**Comment by MorrisJobke** [comment]
Fixed in #6665
